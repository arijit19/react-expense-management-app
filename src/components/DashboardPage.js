import React from 'react';
import ExpenseList from './ExpenseList.js';
import ExpenseFilters from './ExpenseFilters.js'


const Dashboard = ()=>(
   <div>
        <ExpenseFilters/>
        <ExpenseList/>
   </div>
);

export default Dashboard;