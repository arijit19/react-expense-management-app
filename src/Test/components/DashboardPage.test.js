import React from 'react';
import {shallow} from 'enzyme';

import DashboardPage from '../../components/DashboardPage.js'

test('Testing Dashboard page component', ()=>{
    const wrapper = shallow(<DashboardPage/>);
    expect(wrapper).toMatchSnapshot();
})