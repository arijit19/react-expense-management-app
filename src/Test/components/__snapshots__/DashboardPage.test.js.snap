// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Dashboard page component 1`] = `
<div>
  <Connect(ExpenseFilters) />
  <Connect(ExpenseList) />
</div>
`;
